### ISSUE TITLE
Accessibility: Content not included in landmarks - Breadcrumbs

### ISSUE THREAD

**Issue by kevinkidwell** [cul-it/exhibits-library-cornell-edu#439]
**Description**
All content on a webpage should be inside landmarks. This allows screen readers to understand the purpose of different areas of the page at a glance based on the defined roles that these landmarks have.

**Solution**
Option 1: The `breadcrumbs-container` div should have `role="navigation"` and `aria-label="Breadcrumbs"` added to it.
Option 2: Change `<div class="container breadcrumbs-container">` to `<nav class="container breadcrumbs-container" aria-label="Breadcrumbs">`. This would make use of native HTML5 landmark elements instead of relying on the role attribute.

**Comment by mhk33** [comment]
Fixed in dev/3.0 upgrade
